_episodes/17-parallel.md

```diff
@@ -38,8 +38,14 @@ Move into the extracted directory, then use the Package Installer for Python,
 or `pip`, to install it in your ("user") home directory:
 
 ```
-{{ site.remote.prompt }} cd amdahl
+{{ site.remote.prompt }} wget -O amdahl.tar.gz https://github.com/hpc-carpentry/amdahl/tarball/main
+{{ site.remote.prompt }} tar -xvzf amdahl.tar.gz
+{{ site.remote.prompt }} cd hpc-carpentry-amdahl-46c9b4b
```

Comment:
Wouldn't the hash at the end change every time `amdahl`'s repository gets updated? It hasn't change in two years though....

Reply:
Yep that's true- but it looks like there's not a release for the latest version of that repo, and since these are the instructor notes and it's clear what the intent is I'll let it be.